rpcs3/rpcs3qt/tooltips.h

```diff
@@ -60,10 +60,10 @@ class Tooltips : public QObject
 		const QString ppu_llvm                  = tr("Recompiles the game's executable once before running it for the first time.\nThis is by far the fastest option and should always be used.\nShould you face compatibility issues, fall back to one of the Interpreters and retry.\nIf unsure, use this option.");
 		const QString spu_precise               = tr("This is extremely slow but may fix broken graphics in some games.");
 		const QString spu_fast                  = tr("This is slower than the SPU Recompiler but significantly faster than the precise interpreter.\nGames rarely need this however.");
-		const QString spu_asmjit                = tr("This is the fast option with very good compatibility.\nIf unsure, use this option.");
-		const QString spu_llvm                  = tr("This is the fastest option with very good compatibility.\nRecompiles the game's SPU LLVM cache before running which adds extra start-up time.\nIf you experience issues, use the ASMJIT Recompiler.");
+		const QString spu_asmjit                = tr("Recompiles the game's SPU LLVM cache on the go\nThis is the fast option with very good compatibility.\nIf unsure, use this option.");
```

Comment:
If it uses ASMJIT, it doesn't use LLVM. Both solutions use cache and load them before booting